assets/main.js

```diff
@@ -0,0 +1,134 @@
+class Book {
+  constructor(author, title, pages, read = 'Unread') {
+    this.author = author;
+    this.title = title;
+    this.pages = pages;
+    this.read = read;
+  }
+}
+
+const getBooksFromLocal = () => JSON.parse(localStorage.getItem('books')) || [];
+
+function addBookToLocal(book) {
+  const books = getBooksFromLocal();
+  books.push(book);
+
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function findBook(title, author, pages) {
+  const books = getBooksFromLocal();
+  let i = -1;
+
+  books.forEach((b, index) => {
+    if (b.title === title && b.author === author && b.pages === pages) {
+      i = index;
+    }
+  });
+
+  return i;
+}
+
+function removeBookFromLocal(title, author, pages) {
+  const books = getBooksFromLocal();
+  const index = findBook(title, author, pages);
+  if (index >= 0) { books.splice(index, 1); }
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function addBook(book) {
+  const table = document.querySelector('#book-list');
+  const row = document.createElement('tr');
+  const {
+    author, title, pages, read,
+  } = book;
+
+  row.innerHTML = `
+    <td>${author}</td>
+    <td>${title}</td>
+    <td>${pages}</td>
+    <td><button class ="btn btn-info read">${read}</btn></td>
+    <td><button class ="btn btn-danger delete">X</btn></td>
+  `;
+
+  table.appendChild(row);
+}
+
+function displayBooks() {
+  const myBookStore = getBooksFromLocal();
+
+  myBookStore.forEach((book) => {
+    addBook(book);
+  });
+}
+
+function removeBook(element) {
+  if (element.classList.contains('delete')) {
+    element.parentElement.parentElement.remove();
+  }
+}
+
+function toggleClasses() {
+  document.getElementById('book-form').classList.toggle('d-none');
+  document.getElementById('hidden').classList.toggle('d-none');
+}
+
+document.addEventListener('DOMContentLoaded', displayBooks());
+
+document.getElementById('book-form').addEventListener('submit', e => {
+  e.preventDefault();
+  const author = e.target.author.value;
+  const title = e.target.title.value;
+  const pages = e.target.pages.value;
+  const read = (e.target.read.checked) ? 'Read' : 'Unread';
+
+  const book = new Book(author, title, pages, read);
+
+  addBook(book);
```

Comment:
- [ ] **[OPTIONAL]** The form used to create a book should have validations, you should not be able to create an empty book.

I suggest that you create a new function `validateBook()` and run it before you add the book.